src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $background;
+
+  &__stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 0;
+    padding-top: 16px;
+  }
+
+  &__img {
+    margin: 32px 19px 40px;
+    width: 160px;
+    height: 134px;
+    display: block;
+    object-fit: cover;
+  }
+
+  &__text {
+    padding-left: $padding-left;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: $line-height;
+    color: $color-text;
+    margin: 0;
+  }
+}
+
+.text--one {
+  padding-top: 4px;
+  padding-left: $padding-left;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $color-text-next;
+  margin: 0;
+  width: 98px;
+  height: 14px;
+}
+
+.card__reviews {
+  padding-right: $padding-left;
+  color: $color-text;
+  display: flex;
+  text-align: right;
+  align-items: center;
+  justify-content: right;
+  font-size: 10px;
+  line-height: 14px;
+  margin: 0;
+  padding-left: 13px;
+  width: 53px;
+  height: 16px;
+}
+
+.text--four {
+  color: $color-text;
+  text-align: right;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: $line-height;
+  padding-right: 16px;
+  margin: 0;
+}
+
+.text--three {
+  color: $color-text-next;
+  font-size: 12px;
+  line-height: $line-height;
+  padding-left: $padding-left;
+  margin: 0;
+}
+
+.card__button {
+  margin: 16px;
+  font-weight: 700;
+  width: 166px;
+  height: 40px;
+  background-color: $color-border;
+  color: white;
+  line-height: 40px;
+  border: none;
+  border-radius: 5px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 12px;
+}
+
+.card__button:hover {
+  cursor: pointer;
+  border-radius: 5px;
+  border: 1px solid $color-border;
```

Comment:
The variable `$color-border` is used here, but it's not defined within this file. Ensure that it is defined in an imported file or within this file to avoid compilation errors.